src/PublicApiGeneratorTests/Method_modifiers.cs

```diff
@@ -20,15 +48,57 @@ public class ClassWithStaticMethod
         }
 
         [Fact]
-        public void Should_output_abstract_modifier()
+        public void Should_output_protected_static_modifier()
         {
-            AssertPublicApi<ClassWithAbstractMethod>(
-@"namespace PublicApiGeneratorTests.Examples
+            AssertPublicApi<ClassWithProtectedStaticMethod>(
+                @"namespace PublicApiGeneratorTests.Examples
 {
-    public abstract class ClassWithAbstractMethod
+    public class ClassWithProtectedStaticMethod
     {
-        protected ClassWithAbstractMethod() { }
-        public abstract void DoSomething();
+        public ClassWithProtectedStaticMethod() { }
+        protected static void DoSomething() { }
+    }
+}");
+        }
+
+        [Fact]
+        public void Should_output_static_new_modifier()
+        {
+            AssertPublicApi<ClassWithStaticNewMethod>(
+                @"namespace PublicApiGeneratorTests.Examples
+{
+    public class ClassWithStaticNewMethod : PublicApiGeneratorTests.Examples.ClassWithStaticMethod
+    {
+        public ClassWithStaticNewMethod() { }
+        public new static void DoSomething() { }
```

Comment:
I was thinking this was typically `static new` but I'm not finding a canonical list, so...?

Reply:
Honestly I used ReSharper suggestion but will double check

Reply:
Sounds good as-is 👍

Reply:
Seems microsoft mostly uses "static new"

https://github.com/dotnet/corefx/search?q=%22static+new%22&unscoped_q=%22static+new%22

I've asked on twitter. In the meantime let's pull this in as is. We can change it later once we know more